content/adapters/pubnub.md

```diff
@@ -25,6 +25,7 @@ pubnub = Pubnub.new(
   :subscribe_key => api_key,
   :publish_key   => api_key,
   :origin        => 'pubnub.ably.io',
+  :origins_pool  => ["pubnub.ably.io"],
```

Comment:
What type is origins_pool?

Reply:
looks like an array of strings to me

Reply:
would ['pubnub.ably.io'] make sense from a consistency point?

Reply:
sure